cms/service/ScoringService.py

```diff
@@ -30,6 +30,7 @@
 from cms import ServiceCoord, config
 from cms.db import SessionGen, Submission, Dataset, get_submission_results
 from cms.io import Executor, TriggeredService, rpc_method
+from cmscommon import datetime
```

Comment:
I think we don't need to import this explicitly, because `make_datetime` has already been imported below?